pages/design-tokens/typesetting/font-family.md

```diff
@@ -237,3 +239,41 @@ Your context and coding style determine how you access USWDS family tokens in co
     <div class="grid-col-5">.font-<code>body</code>-<code>2xl</code>;</div>
   </div>
 </div>
+
+## Setting custom fonts
+
+If you want to use fonts outside of the ones mentioned in USWDS Available Fonts, you have a few options. The complexity of importing and setting other fonts depends on how you’re including that font.
+
+### Option 1: Import a font from the web
+
+If you’re importing a new font from an open source font web directory,often you’ll be including JavaScript at the top of your file, and that loads the font and associates it with a display name. To make sure that happens, you’ll need to make a typeface token that uses that display name and associates a fallback font stack with it. Here’s how to make the customizations to your code:
+
+1. Define: Tell USWDS about the font you’re using by defining a new typeface token in your settings. In the code example, we are using the font Lato.
+
+    ```sass
+    $theme-typeface-tokens: (
+      "lato": (
+        "display-name": "Lato Web", // or other font
+        "cap-height": 364px,        // the default, leave it for now
+        "stack": "Helvetica Neue, Helvetica, Roboto, Arial, sans-serif", // or whatever stack you want
+      ),
+    ),
+    ```
+
+2. Associate new token with type: Then associate your new "lato" token with the "sans" font type:
+
+    ```sass
+    $theme-font-type-sans: "lato",
+    ```
+
+3. It works! Now everything that uses the "sans" token will use Lato. The CSS will now include something like:
+
+    ```sass
+    font-family: "Lato Web", "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif
+    ```
+
+For more information on how USWDS Typography works,  [Typography settings]({{ site.baseurl }}/design-tokens/typesetting){:.text-ink.text-bold}.
+
+### Option 2: Change default font settings
+
+USWDS settings variables tell the design system how to build. If you want to change the default font settins in uswds-core, see the instructions in [Configuring custom USWDS settings]({{ site.baseurl }}/documentation/settings/#configuring-custom-uswds-settings)
```

Comment:
Typo fix

```suggestion
USWDS settings variables tell the design system how to build. If you want to change the default font settings in `uswds-core`, see the instructions in [Configuring custom USWDS settings]({{ site.baseurl }}/documentation/settings/#configuring-custom-uswds-settings)
```

Reply:
@mejiaj @bonnieAcameron 
Option 2 feels vague or unfinished to me. Can you help me understand what this is trying to say? What is the difference between option 1 and option 2?

Reply:
Option 2 is changing which pre-loaded fonts load in the DS (I think!), where as Option 1 is how to load a web-based custom font. Do you have any thoughts about how I could make this more clear?

Reply:
Maybe: 

USWDS settings variables tell the design system how to build. If you want to change which fonts apply by default in `uswds-core`, see the instructions in [Configuring custom USWDS settings]({{ site.baseurl }}/documentation/settings/#configuring-custom-uswds-settings).

Reply:
We could also flip the order and the title. How I understand it is _how to change the font_:

1. Change default to another USWDS typeface
2. Use the instructions to use a font that's not in USWDS

Reply:
@mejiaj @bonnieAcameron Is "Option 2" covered by the "[Customizing family tokens](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/bc-custom-font-guidance/design-tokens/typesetting/font-family/#customizing-family-tokens)" content higher up on that page? 

If so, could we remove Option 1/Option 2 language and rename this new section to more specifically state that this applies to non-USWDS fonts?  Something like "Creating a custom token with non-USWDS fonts" or "Using non-USWDS fonts"?

Apologies if I am not understanding.